stubs/redis/redis/commands/core.pyi

```diff
@@ -28,27 +33,132 @@ class CoreCommands(Generic[_StrType]):
         reset: bool = ...,
         reset_keys: bool = ...,
         reset_passwords: bool = ...,
+        **kwargs,
     ) -> bool: ...
-    def acl_users(self) -> list[str]: ...
-    def acl_whoami(self) -> str: ...
+    def acl_users(self, **kwargs) -> list[str]: ...
+    def acl_whoami(self, **kwargs) -> str: ...
+
+class ManagementCommands:
+    def bgrewriteaof(self, **kwargs): ...
+    def bgsave(self, schedule: bool = ..., **kwargs): ...
+    def role(self): ...
+    def client_kill(self, address: str, **kwargs) -> bool: ...
+    def client_kill_filter(
+        self,
+        _id: Any | None = ...,
+        _type: Any | None = ...,
+        addr: Any | None = ...,
+        skipme: Any | None = ...,
+        laddr: Any | None = ...,
+        user: Any | None = ...,
+        **kwargs,
+    ): ...
+    def client_info(self, **kwargs): ...
+    def client_list(self, _type: str | None = ..., client_id: list[str] = ..., **kwargs) -> list[dict[str, str]]: ...
+    def client_getname(self, **kwargs) -> str | None: ...
```

Comment:
Can the methods in this class also return `bytes`?

Reply:
See above.